packages/react-core/src/components/Popover/PopoverHeader.tsx

```diff
@@ -31,23 +30,17 @@ export const PopoverHeader: React.FunctionComponent<PopoverHeaderProps> = ({
   id,
   alertSeverityScreenReaderText,
   ...props
-}: PopoverHeaderProps) => {
-  const HeadingLevel = titleHeadingLevel;
-
-  return icon || alertSeverityVariant ? (
-    <header className={css('pf-c-popover__header', className)} id={id} {...props}>
-      <HeadingLevel className={css(styles.popoverTitle)}>
-        {icon && <PopoverHeaderIcon>{icon}</PopoverHeaderIcon>}
+}: PopoverHeaderProps) => (
+  <header className={css(styles.popoverHeader, className)} {...props}>
+    <div className={css(styles.popoverTitle)}>
+      {icon && <PopoverHeaderIcon>{icon}</PopoverHeaderIcon>}
+      <PopoverHeaderText headingLevel={titleHeadingLevel} id={id}>
```

Comment:
This is fine but should it be `<PopoverHeaderTitleText>`? That will match the core class. It's named that way to try and avoid naming conflicts if we introduce something like a subtitle in the header.